### PR TITLE
chore: changelog for SDK 9.0.0

### PR DESCRIPTION
**Pull request by inckie**
### Detailed readable description

Changelog for SDK 9.0.0

### Checklist
- [x] I've provided a readable title for the PR
- [x] I've provided a detailed description
- [x] The PR is scoped to a single task/feature
- [ ] I've included relevant tests (if tests are not included please provide the reason why they aren't)

- [x] My PR is not introducing a UI change
- [ ] My PR is introducing UI changes and I provided all screenshots in the PR description

- [x] My PR is a part of a new feature or a feature improvement
    - [ ] I've provided a link in the description to the relevant card in the Zapp cycle feature rollout Trello board.
- [ ] My PR is a bug fix
    - [ ] I provided a link in the description to the relevant Jira ticket  or provided the following in the PR description:
        - steps to reproduce a bug
        - expected result

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,92 @@
...
+- Legacy analytics parameters removed [#1515](https://github.com/applicaster/applicaster-android-sdk/pull/1515)
+- Player analytics events aligned with new requirements [#1513](https://github.com/applicaster/applicaster-android-sdk/pull/1513), [#1509](https://github.com/applicaster/applicaster-android-sdk/pull/1509), [#1506](https://github.com/applicaster/applicaster-android-sdk/pull/1506)
+- Initial configuration loading reworked [#1511](https://github.com/applicaster/applicaster-android-sdk/pull/1511), [#1504](https://github.com/applicaster/applicaster-android-sdk/pull/1504), [#1501](https://github.com/applicaster/applicaster-android-sdk/pull/1501)
+- Purchase analytics event router added [#1505](https://github.com/applicaster/applicaster-android-sdk/pull/1505), [#1502](https://github.com/applicaster/applicaster-android-sdk/pull/1502)
+- HTTP request method, request and response headers are mow logged [#1500](https://github.com/applicaster/applicaster-android-sdk/pull/1500)
+
+**Changes**
+- Advertising Id collection moved to plugin to comply with new Google Play requirements [#1510](https://github.com/applicaster/applicaster-android-sdk/pull/1510)
```
Comment (ranmeirman) [inline CHANGELOG.md:44]: Let's move to breaking changes. I would also remove completely the sdk separation, the reader does not really care if it is sdk or platform

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,92 @@
 # Changelog
 
+
+## 9.0.0 (January 26, 2022)
```
Comment (ranmeirman) [inline CHANGELOG.md:4]: Date is wrong, it's August:)

**Review by ranmeirman — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (inckie) [reply CHANGELOG.md]: Indeed, fixed

File: CHANGELOG.md (hunk 1 above)
Comment (inckie) [reply CHANGELOG.md]: Moved and merged sections

**Review by ranmeirman — APPROVED** [review]

**Pull request merged**